log/log.go

```diff
@@ -179,6 +182,12 @@ func ErrorC(context string, err error, data Data) {
 	Event("error", context, data)
 }
 
+// ErrorCtx is a structured info message for go context
+func ErrorCtx(ctx context.Context, err error, data Data) {
+	context := requestID.Get(ctx)
+	ErrorC(context, err, data)
+}
+
 // ErrorR is a structured error message for a request
 func ErrorR(req *http.Request, err error, data Data) {
 	ErrorC(Context(req), err, data)
```

Comment:
consider renaming the "Context" function since it has very little to do with a go context.Context

Reply:
yeah, how about `correlationId`